src/services/book.service.ts

```diff
@@ -1,32 +1,34 @@
-const fs = require('fs');
-const path = require('path');
+import fs from "fs";
+import path from "path";
 
-class Book {
-    static dbPath = path.join(__dirname, '..', 'data', 'mock-data.json');
+export default class Book {
+    static dbPath: string = path.join(__dirname, '..', 'data', 'mock-data.json');
 
-    constructor(title, author) {
+    id?: string;
+
+    constructor(public title: string, public author: string) {
         this.title = title;
         this.author = author;
     }
 
-    static getAllBooks() {
+    static getAllBooks(): Promise<Book[]> {
```

Comment:
I receive 500 Error on getting all books. I think it is connected with tsconfig file that is not copied mock.data.json file into build folder